rust/bin-shared/src/known_dirs/macos.rs

```diff
@@ -1,25 +1,59 @@
+use crate::BUNDLE_ID;
 use std::path::PathBuf;
 
+/// Returns the user's home directory
+fn home_dir() -> Option<PathBuf> {
+    std::env::var_os("HOME").map(PathBuf::from)
+}
+
+/// Path for Tunnel service config that the Tunnel service can write
+#[expect(clippy::unnecessary_wraps)] // Signature must match Windows
 pub fn tunnel_service_config() -> Option<PathBuf> {
-    unimplemented!()
+    Some(
+        PathBuf::from("/Library/Application Support")
```

Comment:
I think some of these could potentially also be implemented using the `dirs` library we already depend on.